### ISSUE TITLE
Traducción del curso Mainframe COBOL con VSCode

### ISSUE THREAD

**Issue by DanielSReynoso** [openmainframeproject/cobol-programming-course#204]
Traducción del Ingles al Castellano del curso Mainframe COBOL con VSCode

[1 - Configurando VSCode.pdf](https://github.com/openmainframeproject/cobol-programming-course/files/5853173/1.-.Configurando.VSCode.pdf)
[2 - Ahora si -- CoBOL!!.pdf](https://github.com/openmainframeproject/cobol-programming-course/files/5853175/2.-.Ahora.si.--.CoBOL.pdf)

**Comment by MikeBauerCA** [comment]
@DanielSReynoso thank you very much for providing these translations. With your approval, I'm considering adding these documents to our course release pending review and discussion with others. Would you be interested in helping address any issues the community may find going forward?

**Comment by DanielSReynoso** [comment]
It will be a pleasure and an honor to collaborate in what is within my power. Hug everyone

Enviado desde mi smartphone Sony Xperia™

---- MikeBauerCA escribió ----


@DanielSReynoso<https://github.com/DanielSReynoso> thank you very much for providing these translations. With your approval, I'm considering adding these documents to our course release pending review and discussion with others. Would you be interested in helping address any issues the community may find going forward?

**Comment by DanielSReynoso** [comment]
Of course, you can count on my collaboration, and you can definitely use this work for the benefit of the Spanish-speaking community. At the moment I'm studying hard because I hope to get to work again as a cobol programmer. But I hope this week to review the translation to improve the understanding of it considering that there are many technical terms that must be kept in their original language. Finally, I reiterate that you can count on my collaboration. A hug for everyone

Daniel S. Reynoso
danielsilvio.reynoso@bue.edu.ar
+54 911 2817 7965
________________________________
De: MikeBauerCA <notifications@github.com>
Enviado: viernes, 22 de enero de 2021 11:34
Para: openmainframeproject/cobol-programming-course <cobol-programming-course@noreply.github.com>
Cc: Daniel Silvio Reynoso <dsreynoso@hotmail.com>; Mention <mention@noreply.github.com>
Asunto: Re: [openmainframeproject/cobol-programming-course] Traducción del curso Mainframe COBOL con VSCode (#204)


@DanielSReynoso<https://github.com/DanielSReynoso> thank you very much for providing these translations. With your approval, I'm considering adding these documents to our course release pending review and discussion with others. Would you be interested in helping address any issues the community may find going forward?

**Comment by MikeBauerCA** [comment]
@DanielSReynoso how would you recommend others contribute to these documents? In the course, typically we collaborate on markdown files within the repository and generate a PDF from them. If you had these translations in markdown format, we could do the same.

**Comment by tanto259** [comment]
Just my two cents here: I would agree with the markdown thing, that way we can preserve an identical format with the provided PDF. A concern would be versioning, maybe we can add a special Translation Contributor page with the translator details (if they choose to make it public), and the version they are translating (that way people can know which version they are using).

Also in relation to that, I would say we need to indicate the version of the course for each PDF we publicize.

**Comment by MikeBauerCA** [comment]
I like that approach. We could link to available translations from the README and also include those details in the release notes. If they are available in markdown, we can handle the collaboration here and manage publishing the PDFs. If not, we just need to find another way to collaborate on such documents.

> Also in relation to that, I would say we need to indicate the version of the course for each PDF we publicize.

Are you suggesting also including the version in the file names - https://github.com/openmainframeproject/cobol-programming-course/releases/tag/2.2.0

For example, maybe change `COBOL-Programming-Course-1-Getting-Started.pdf` to `COBOL-Programming-Course-1-Getting-Started-v2.2.0.pdf`?

**Comment by tanto259** [comment]
> Are you suggesting also including the version in the file names - https://github.com/openmainframeproject/cobol-programming-course/releases/tag/2.2.0
> 
> For example, maybe change `COBOL-Programming-Course-1-Getting-Started.pdf` to `COBOL-Programming-Course-1-Getting-Started-v2.2.0.pdf`?
> 

That and also somewhere inside the first few pages. Just in case someone re-distribute the PDFs elsewhere, they can know if they have outdated version.

**Comment by Zawng** [comment]
Daniel, tienes una versión en markdown de estos archivos? me gustaría contribuir.

**Comment by MikeBauerCA** [comment]
@DanielSReynoso

**Comment by orion160** [comment]
Hi, I could write a spanish translation.
Ill try to submit a PR for version 3.0 for the course
